SUPPORT.md

```diff
@@ -40,18 +43,73 @@ pkg-fmt = "zip"
 
 ### Defaults
 
-By default `binstall` is setup to work with github releases, and expects to find:
+By default, `binstall` will try all supported package format and would have `bin-dir` set to
+`"{ name }-{ target }-v{ version }/{ bin }{ binary-ext }"` (where `bin` is the cargo binary name and
+`binary-ext` is `.exe` on windows and empty on other platforms).
+
+The default value for `pkg-url` will depend on the repository of the package.
+
+It is setup to work with github releases, gitlab releases, bitbucket downloads
+and source forge downloads.
+
+#### Github
+
+For github, the `pkg-url` is set to
+
+```rust
+"{ repo }/releases/download/v{ version }/{ name }-{ target }-v{ version }.{ archive-format }"
+```
 
 - an archive named `{ name }-{ target }-v{ version }.{ archive-format }`
   - so that this does not overwrite different targets or versions when manually downloaded
 - located at `{ repo }/releases/download/v{ version }/`
   - compatible with github tags / releases
 - containing a folder named `{ name }-{ target }-v{ version }`
   - so that prior binary files are not overwritten when manually executing `tar -xvf ...`
-- containing binary files in the form `{ bin }{ binary-ext }` (where `bin` is the cargo binary name and `binary-ext` is `.exe` on windows and empty on other platforms)
 
 If your package already uses this approach, you shouldn't need to set anything.
 
+#### Gitlab
```

Comment:
minor: can we have consistent casing/form for all mentions of these names? ideally using the brand's preferred form, ie `GitHub`, `GitLab`, `Bitbucket`, `SourceForge`

Reply:
Done!